trajopt_ifopt/cmake/trajopt_ifopt-config.cmake.in

```diff
@@ -0,0 +1,17 @@
+@PACKAGE_INIT@
+
+set(@PROJECT_NAME@_FOUND ON)
+set_and_check(@PROJECT_NAME@_INCLUDE_DIRS "${PACKAGE_PREFIX_DIR}/include")
+set_and_check(@PROJECT_NAME@_LIBRARY_DIRS "${PACKAGE_PREFIX_DIR}/lib")
+
+include(CMakeFindDependencyMacro)
+find_dependency(Eigen3)
+if(${CMAKE_VERSION} VERSION_LESS "3.10.0")
+    find_package(Boost COMPONENTS system python thread program_options)
+else()
+    find_dependency(Boost COMPONENTS system python thread program_options)
+endif()
+find_dependency(console_bridge)
+find_dependency(ifopt)
+
```

Comment:
Need to update find_dependencies for the find packages in the cmake.

find_package(trajopt_utils)
find_package(trajopt)
find_package(console_bridge REQUIRED)
find_package(Eigen3 REQUIRED)
find_package(ifopt REQUIRED)
find_package(tesseract REQUIRED)